compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/JvmGeneratorExtensions.kt

```diff
@@ -74,6 +80,17 @@ class JvmGeneratorExtensions(private val generateFacades: Boolean = true) : Gene
         else
             null
 
+    override fun computeParameterName(descriptor: ParameterDescriptor): Name {
```

Comment:
Minor, just curious: why is it JVM-specific?

Reply:
Honestly I'm not sure if it should be specific to JVM or not. The mangling only happens in codegen for the old JVM backend, so I took that as a cue that it should be specific to JVM.